### ISSUE TITLE
Spawning Mobs Inside of ASync Does Not Work

### ISSUE THREAD

**Issue by MinecraftMan1013number2** [Olyno/skent#32]
## Describe your problem

Spawning mobs does not work inside of something async.

## Versions

**Skript:** 2.6.1
**Server:** Paper version git-Paper-"b8fa41e" (MC: 1.18.2)
**Skent:** 3.1.0
![image](https://user-images.githubusercontent.com/70504198/167230724-76f7bcb4-be32-42a2-84f2-cf818a64ece2.png)

## Code

```vb
function RenameAndSpawn(oldPath: text, newName: text):
	set {_path} to file path "%{_oldPath}%"
	if {_path} is not missing:
		set {_p} to location of "MinecraftMan1013" parsed as a player
		rename file path "%{_oldPath}%" to "%{_newName}%":
			spawn 1 creeper at {_p}
			set {_z} to last spawned creeper
			set display name of {_z} to "Test Dummie"
			broadcast "spawned"
		broadcast "done"
	else:
		broadcast "requested file missing"
```
Result:
![image](https://user-images.githubusercontent.com/70504198/167230691-34172c3f-3165-4894-8448-5a458124968e.png)
No mob was spawned at my location though.

## Error

It did this once (I was using armor stands at the time), but not for my other tests.
[https://www.toptal.com/developers/hastebin/zolifopili](https://www.toptal.com/developers/hastebin/zolifopili)

**Comment by Olyno** [comment]
Hi 👋🏻 
Indeed, this is probably due to the async problem that is not possible to fix at the moment (at least not easily)

**Comment by MinecraftMan1013number2** [comment]
Dang, that stinks. Is there any way around this problem?

**Comment by Olyno** [comment]
Except for spending a lot of time on it in order to have a good method and make it work properly, not really. You'll have to wait and be patient

**Comment by MinecraftMan1013number2** [comment]
Ok, thanks.

**Comment by Olyno** [comment]
Should be fixed in the next release (3.2.0). Feel free to reopen the issue if you still have the issue.
